packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -62,6 +62,7 @@ export const GridRootStyles = styled('div', {
     height: '100%',
     display: 'flex',
     flexDirection: 'column',
+    colorScheme: theme.palette.mode,
```

Comment:
We can leave a note for developers to upgrade to MUI Core 5.1 or use https://mui.com/components/css-baseline/#scrollbars

```suggestion
```

Reply:
Is it possible that DataGrid is used for itself (not in MUI Core)? If yes, we should provide an explanation of how to set dark mode for DataGrid itself.

Reply:
I guess you meant `@mui/material`? It's a peer dep so I think it's ok to have a case like that.

Reply:
> Is it possible that DataGrid is used for itself (not in MUI Core)?

Yes, but `@mui/material` will always be installed. What changes is that they may not add the `CssBaseline`. In this case, other parts of the their applications will also look wrong without setting `color-scheme`. Overriding the color scheme only for the DataGrid